src/main/webapp/view/transcript/transcript.js

```diff
@@ -87,6 +93,17 @@ export class TranscriptLineElement extends HTMLElement {
     return hrElement;
   }
 
+  /**
+   * Creates a stylized comment indicator element.
```

Comment:
I was thinking the comment indicator could start out invisible so that is doesn't appear empty until the comments are counted. WDYT?

Reply:
Ah, the comment indicators are initially invisible. I set the visibility in the css.
https://github.com/googleinterns/zoomtube/pull/306/files#diff-52940dbcf82096e65b2848e5b8b28e41R128
Should I mention that they are invisible in the method header?

Reply:
I would expand on this method.

/**
 * Creates an invisible stylized comment indicator element.
 *
 * <p> Once the element is populated then it becomes visible.
 */

Reply:
gotcha, I update it to be that. Thanks!